src/index.html

```diff
@@ -17,11 +18,368 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
     />
   </head>
+  <!-- #endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <div class="header__logo">
+        <a href="/">
+          <img
+            src="../src/images/logo.png"
+            alt="Site logo"
+          />
+        </a>
+      </div>
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              class="header__link header__link--active is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="header__item"
+            data-qa="nav-hover"
+          >
+            <a
+              class="header__link"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="/src/images/imac.jpeg"
+          alt="Product image"
+        />
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
```

Comment:
Would be better to use semantic tags for such case, so `ul` for wrapper, `li` for items
```suggestion
          <ul class="stars">
            <li class="stars__star stars__star--active"></li>
            <li class="stars__star stars__star--active"></li>
            <li class="stars__star stars__star--active"></li>
```